src/radical/pilot/raptor/master.py

```diff
@@ -303,6 +304,7 @@ def submit_workers(self, descr: Dict[str, Union[str, int]],
                 ru.write_json(cfg, fname)
 
                 td = dict()
```

Comment:
this line could be updated as well (as we discussed earlier): `td = dict(descr)`
and only several lines below would be needed
```python
                td['mode']             = RAPTOR_WORKER
                td['cpu_process_type'] = rpc.MPI
                td['cpu_thread_type']  = rpc.POSIX

                td['executable'] = 'python3'
                td['arguments']  = [...
```

Reply:
Probably right, but out of scope for this PR?